docs/safemode.md

```diff
@@ -0,0 +1,55 @@
+# Safemode
+
+Safemode represents a number of safety nets we have implemented into the chaos controller to help new and experienced users feel more confident deploying new disruptions to their environments.
+
+The Chaos Controller can be scary to use in production environments, but we learn a lot more running chaos experiments in our production environments. Considering this, having safety nets in place makes the entire process of running chaos experiments in high value environments a little safer.
+
+Safemoe will be added true by default and will require manual disabling of safety nets in order to bypass. In the disruption schema we have `unsafeMode` which represents ways to remove the safety nets.
+`disableAll` is self-explainatory while the other disables in the unsafe schema represent each safety net which you may disable one by one.
+Please take a look at the example below to see how to use the `unsafeMode` parameter.
+
+## Ignoring Safety Nets
+
+Because the list of safety nets to be implemented will grow in the future, there will surely be overlap with safety nets which will make it difficult for a user who is confident a specific safety net is not necessary but unsure if others will be.
+Therefore the controller allows for you to disable specific safety nets in the Safemode Spec. Checkout out example below to see how to remove certain safety nets.
+Keep in mind that all safety nets are turned on by default.
+
+### Safety Nets
+
+| Safety Net                                                | Type | Description                                                                                                                                               | IgnoreName                 |
```

Comment:
Could the language under `Description` be a bit more consistent for each?

Reply:
I will try and be more consistent in the language in the next commitment, please let me know if it is still not consistent.